vars.tf

```diff
@@ -40,7 +40,7 @@ variable "backup_path" {
 }
 
 variable "docker_tag" {
-  default = "latest"
+  default = "0.1.6"
```

Comment:
I would prefer to leave this as `latest` and change it only on [HCP Terraform](https://app.terraform.io/session)